src/modules/elements/action-dashboard/style.pcss

```diff
@@ -27,6 +27,12 @@
 	flex: none;
 	margin-right: 25px;
 
+	.tribe-editor__link {
```

Comment:
These styles should go in the action button instead, not in the action dashboard. Also need styles for `:focus`.

Reply:
sure, no problem with the specificity. In the action button styles file, you can add another class specific for that style. but main thing is where the style lives so we're not putting styles for the same thing in different places.